README.md

```diff
@@ -637,60 +982,111 @@ jest.fn(() => {
 });
 ```
 
-## `require` Extensions
+## The `jest` object
 
-### `require.requireActual(moduleName)`
+### `jest.clearAllTimers()`
+Removes any pending timers from the timer system.
 
-Returns the actual module instead of a mock, bypassing all checks on whether the
-module should receive a mock implementation or not.
+This means, if any timers have been scheduled (but have not yet executed), they will be cleared and will never have the opportunity to execute in the future.
 
-### `require.requireMock(moduleName)`
+### `jest.disableAutomock()`
+Disables automatic mocking in the module loader.
 
-Returns a mock module instead of the actual module, bypassing all checks on
-whether the module should be required normally or not.
+After this method is called, all `require()`s will return the real versions of each module (rather than a mocked version).
 
-## Property testing
+This is usually useful when you have a scenario where the number of dependencies you want to mock is far less than the number of dependencies that you don't. For example, if you're writing a test for a module that uses a large number of dependencies that can be reasonably classified as "implementation details" of the module, then you likely do not want to mock them.
 
-Jest supports property testing with the
-[testcheck-js](https://github.com/leebyron/testcheck-js) library. The API is
-the same as that of [jasmine-check](https://github.com/leebyron/jasmine-check):
+Examples of dependencies that might be considered "implementation details" are things ranging from language built-ins (e.g. Array.prototype methods) to highly common utility methods (e.g. underscore/lo-dash, array utilities etc) and entire libraries like React.js.
 
-### `check.it(name, [options], generators, fn)`
-Creates a property test. Test cases will be created by the given `generators`
-and passed as arguments to `fn`. If any test case fails, a shrunken failing
-value will be given in the test output. For example:
+*Note: this method was previously called `autoMockOff`. When using `babel-jest`, calls to `disableAutomock` will automatically be hoisted to the top of the code block. Use `autoMockOff` if you want to explicitly avoid this behavior.*
+
+### `jest.enableAutomock()`
+Re-enables automatic mocking in the module loader.
```

Comment:
Let's change this to "Enables automocking in the module loader" because once automock is off by default, this will enable and not re-enable it :)